Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.8-slim-buster
+
+WORKDIR /app
+
+# Install backend dependencies necessary to compile SCS and other solvers
+RUN apt-get update && apt-get install -y build-essential g++ libgl1-mesa-glx libx11-6
```

```suggestion
RUN apt-get update && apt-get install -y build-essential
```

`build-essential` implies `g++`, the last two should not be necessary

---

Good catch!